Libraries/profiling/src/profiling.cpp

```diff
@@ -21,3 +21,163 @@ bool is_profiling_enabled() {
 
 }  // namespace profiling
 }  // namespace wn
+
+#ifdef TRACY_ENABLE
+static const char* default_allocator_name = "DefaultAllocator";
+
+void* aligned_alloc(std::size_t size, std::align_val_t al) {
```

Comment:
These already exists in https://github.com/WNProject/WNFramework/blob/master/Libraries/WNMemory/inc/allocation.h#L29 if you want/can just use those

Reply:
I wanted to avoid having profiling depend on memory. (that way we can use profiing FROM memory if we ever want to).